tests/integration/ceph_mgr_test.go

```diff
@@ -113,9 +113,26 @@ func (s *CephMgrSuite) TearDownSuite() {
 	s.installer.UninstallRook()
 }
 
-func (s *CephMgrSuite) execute(command []string) (error, string) {
-	orchCommand := append([]string{"orch"}, command...)
-	return s.installer.Execute("ceph", orchCommand, s.namespace)
+func (s *CephMgrSuite) execute(command []string, maxRetries int) (error, string) {
+	tries := 0
+	orchestratorCommand := append([]string{"orch"}, command...)
+	for {
+		err, output := s.installer.Execute("ceph", orchestratorCommand, s.namespace)
+		tries++
+		if err != nil  {
```

Comment:
For readability I'd suggest a structure like this where there is no need for a deeper `else` block if you return in the previous block:
```suggestion
		if err != nil  {
			if maxRetries == 1 {
				return output, err
			}
			if tries == maxRetries {
				return "", fmt.Errorf("max retries(%d) reached, last err: %v", tries, err)
			} 
			logger.Infof("retrying command <<ceph %s>>: last error: %v", command, err)
			continue
		}
		return output, nil
```